integrations/ConvertFlow/utils.js

```diff
@@ -0,0 +1,33 @@
+// default mapping for the events
+const standardEventsListMapping = {
+  cfReady: "CTA Ready",
+  cfView: "CTA Viewed",
+  cfConversion: "CTA Converted",
+  cfCompletion: "CTA Completed",
+  cfSubmit: "CTA Form Submitted",
+  cfAddToCart: "Product Added to Cart",
+  cfClosed: "CTA Closed",
+};
+
+/**
+ * This function is used to make a track Call to rudderstack.
+ * @param {*} standardEventsMap - mapping of events done by the user
+ * @param {*} eventName - standard event name
+ * @param {*} data
+ */
+const makeACall = (standardEventsMap, eventName, data) => {
+  const eventNames = Object.keys(standardEventsMap);
+  let updatedEvent = standardEventsListMapping[eventName];
+  eventNames.forEach((event) => {
+    if (standardEventsMap[event].includes(eventName)) {
```

Comment:
Here stadardEventsMap[event] a string right? Then we can't use includes.

Reply:
updated